### DISCUSSION TITLE
No battery health data after multiple charges

### DISCUSSION THREAD

**Discussion by Ba11in0nABudget** [teslamate-org/teslamate#4133] [Getting started] [answered]
Ive read that you need 2 charges below 95% SOC and longer than 10 minutes in this discussion and from the FAQ https://github.com/teslamate-org/teslamate/discussions/3569

I've had multiple charges now. I am only level 1 charging right now as my level 2 home charger hasn't been installed yet, but I have still done charges from 60-80%. What do i need to get data?

![Screenshot 2024-08-09 165652](https://github.com/user-attachments/assets/c8c05a46-e751-4ef5-9b35-548400ff446d)

**Comment by Ba11in0nABudget** [comment]
![Screenshot 2024-08-09 180447](https://github.com/user-attachments/assets/81b6403f-1df6-4074-8eba-620eb8af2f7b)

Just to add detail on the multiple charging sessions that are correctly being logged and displayed in TeslaMate

**Comment by Ba11in0nABudget** [comment, marked as answer]
A kind person on reddit showed me this previous discussion https://github.com/teslamate-org/teslamate/issues/3524 where someone stated that you need to complete a charge that adds atleast 15kwh before you will get data. 

That turns out to be true. I got my level 2 charger installed and did a charge from 30-80% and now all the data populated. 

So if anyone else goes looking for this info. 

You need multiple charges of which atleast one adds 15kwh to the battery before you get data.

**Reply by arcastro** [reply]
Just wanted to say thank you for documenting the question and answer. This helped me out :)
